webapp/WEB-INF/templates/admin/plugins/document/manage_documents.html

```diff
@@ -377,3 +383,12 @@ <h3 class="box-title">#i18n{document.manage_documents.labelMoreActions}</h3>
 	});
 
 </script>
+
```

Comment:
Maybe add the same check for is_files2doc_plugin_active for this script ?

Reply:
done in commit ae6bd9e